app/src/main/java/com/amaze/filemanager/utils/application/AppConfig.java

```diff
@@ -51,7 +51,10 @@
     private UtilsHandler mUtilsHandler;
 
     private static Handler mApplicationHandler = new Handler();
-    private static HandlerThread sBackgroundHandlerThread = new HandlerThread("app_background");
+    //This caused the IllegalThreadStateException because it is being declared and initialized
```

Comment:
This comment is too specific to an issue in github, only use these type of comments for hacks. If needed, use javadoc explaining the variable.

Reply:
Oh so for future reference comments like this are not good, is it better for me to make these sorts of descriptions in my pull request?

Reply:
Surely